server/api-docs.apib

```diff
@@ -86,13 +86,16 @@ Create completion for card
 - Response 201 (application/json; charset=utf-8)
 
     - Attributes (object)
-        + completedAt (string) - Timestamp of the card completion creation.
+        + card (object)
```

Comment:
@AleksSenkou I think it's better to add this changes in another commit, because current commit message is confusing. I don't understand how it can help to database.

Reply:
@DDKatch sure, will add it as a separate commit message